spec/requests/search_spec.rb

```diff
@@ -0,0 +1,42 @@
+require './spec/spec_helper'
+
+def app
+  SearchController
+end
+
+RSpec.describe 'SearchController' do
```

Comment:
As a refactor, we may be able to take this out and create a separate folder - similar to what Phillip was discussing this morning that allows us to consolidate the checks we're making on the payload